### PR TITLE
add platform 37

### PR DESCRIPTION
**Pull request by avtolstoy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "eslint": "^8.17.0",
     "eslint-config-particle": "^2.5.0",
     "lodash": "^4.17.21",
+    "nise": "^5.1.4",
```
Comment (hugomontero) [inline package.json:46]: Is this dependency required for this PR? `"nise": "^5.1.4",`

**Review by hugomontero — APPROVED** [review]
Related to 👉  https://github.com/particle-iot/device-os-test-runner/compare/master...chore/sc-132686/support-electron-2-platform-in-cli-tools

File: package.json (hunk 1 above)
Comment (avtolstoy) [reply package.json:46]: There is a downstream dependency issue with `fake-timers`, which doesn't work under Node 12. Forcing `nise` to stay at 5.1.4 fixes this. Only affects dev dependencies/tests.

**Review by technobly — COMMENTED** [review]
Tested locally and it appears to work, just need the cloud to recognize it now. 👍🏼

**Pull request merged**